iquip/apps/stage.py

```diff
@@ -336,3 +339,141 @@ def _relativePositiveMove(self):
     def _relativeNegativeMove(self):
         """Relative negative move button is clicked."""
         self.moveBy.emit(-self.relativeBox.value() / 1e3)
+
+
+class StageControllerFrame(QWidget):
+    """Frame for StageControllerApp.
+    
+    Attributes:
+        widgets: Dictionary whose keys are stage names and the values are the
+          corresponding stage widgets.
+    """
+
+    def __init__(
+        self,
+        stages: Dict[str, Dict[str, Any]],
+        parent: Optional[QWidget] = None,
+    ):
+        """Extended.
+        
+        Args:
+            See StageControllerApp.
+        """
+        super().__init__(parent=parent)
+        self.widgets: Dict[str, StageWidget] = {}
+        layout = QGridLayout(self)
+        for name, info in stages.items():
+            widget = StageWidget(self)
+            groupbox = QGroupBox(name, self)
+            groupboxLayout = QHBoxLayout(groupbox)
+            groupboxLayout.addWidget(widget)
```

Comment:
The `groupboxLayout` is used once, so these two lines can be written in just a line:
`QHBoxLayout(groupbox).addWidget(widget)`